src/main/java/org/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package org.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
don't use `@Data` for entities
```suggestion
@Getter
@Setter
```

add `@Table` with name